setup.cfg

```diff
@@ -9,9 +9,14 @@ markers =
 
 [pylint]
 reports = no
-disable = locally-disabled,star-args,consider-using-f-string
+disable = line-too-long,locally-disabled,star-args,consider-using-f-string
 msg-template = {path}:{line}: [{msg_id}({symbol}), {obj}] {msg}
 
 [flake8]
 exclude = .tox,dist,doc,build,*.egg
 max-complexity = 10
+ignore = E501, W503  # line too long and line break before binary operator
+
+[tool.black]
+line-length = 79
```

Comment:
Ok, we set it to 79 here, but why can't the other linters still have this check?

Reply:
Because black is not strict with line-length, it tries to follow it but if it can't because it breaks other formatting rules, it will reformat with a larger line-length, see https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length